src/ImageSharp/Common/Helpers/Numerics.cs

```diff
@@ -748,5 +748,65 @@ public static unsafe void CubeRootOnXYZ(Span<Vector4> vectors)
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static float Lerp(float value1, float value2, float amount)
             => ((value2 - value1) * amount) + value1;
+
+#if SUPPORTS_RUNTIME_INTRINSICS
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static void Accumulate(ref Vector<uint> accumulator, Vector<byte> values)
```

Comment:
If we could get some intellisense docs here that would be great. I like to keep public methods documented on internal types when we're doing fancy stuff like SIMD.

Reply:
Can do! Will also post a bench of base Paeth vs the latest Avx2 version.

Reply:
Ace!